tests/RST212/short_underline.py

```diff
@@ -0,0 +1,22 @@
+"""Print 'Hello world' to the terminal.
+
+Long
+=====
+An overly long underline (here one extra equals sign)
+is considered acceptable.
```

Comment:
Yeah, for whatever reason the reST spec is written to not mind too-long underlines. Probably the assumption is that people will just put huge underlines in, so that they don't have to fiddle with them when they change the header later.

I prefer mine to be perfectly "nice", though, for cosmetic reasons, so the flexibility for longer ones doesn't really help me. 😝

Reply:
Yes, me too - but I'm not about to start adding additional validation into this flake8 plugin.